doc/Type/HyperWhatever.rakudoc

```diff
@@ -33,7 +33,7 @@ being smartmatched.
 
 =head1 Currying
```

Comment:
Was leaving this section header as "Currying"  intentional, or was it just forgotten? Wasn't sure if it made sense to maintain this header text if we're trying to be consistent.